NARFO_BE/ControllerUnitTests/UnitTest1.cs

```diff
@@ -45,7 +45,7 @@ public Tests()
         public async Task GetAllMembers_ShouldReturnAllMembers()
         {
             // Create Member object
-            var newMember = new Members { Id = 1, FirstName = "Mike", LastName = " Jack", Username = "MikeJack", Password = "Password1" };
+            var newMember = new _Member { Id = 1, Firstname = "Mike", SURNAME = " Jack", Username = "MikeJack", Password = "Password1" };
```

Comment:
The surname is in all caps, please make consistent with the other variables

Reply:
it is saved that way in the database  and we importing an existing database so we can't change that now

Reply:
Alright will get that fixed. Thank you

Reply:
Add a migration for it

Reply:
No need to adopt bad naming if you are able to change it for the better

Reply:
@Bongz12 @RetroDavis Why was this resolved? The surname is still all caps.